lib/slack/web/api/too_many_requests_error.rb

```diff
@@ -0,0 +1,21 @@
+module Slack
+  module Web
+    module Api
+      class TooManyRequestsError < ::Faraday::Error
```

Comment:
I think so. It's not uncommon. We can also do a `Slack::Web::Api::Errors::Base` and alias the old one for backward compatibility if you want to, your call.